src/extension/background-script/actions/accounts/add.ts

```diff
@@ -11,10 +13,11 @@ const add = async (message, sender) => {
   const password = state.getState().password;
   const currentAccountId = state.getState().currentAccountId;
 
-  const accountId = Object.keys(accounts).length + 1;
-  console.log(`Created account ${accountId}`);
+  if (!password) return;
```

Comment:
I think we should return some error here. Though this should not happen and the error might not be handled well in the UI, we should still indicate the error by returning a error object.